composer.json

```diff
@@ -30,6 +30,7 @@
         "drupal/core": "8.7.0",
         "drupal/devel": "^2.1",
         "drupal/easy_breadcrumb": "^1.12",
+        "drupal/helper": "^1.3",
```

Comment:
You don't need the helper module anymore remove this please.

Reply:
done.